### PR TITLE
docs(releases): Patching changelogs

### PR DESCRIPTION
**Pull request by dbyron0**
For background, see https://github.com/spinnaker/spinnaker.github.io/issues/2029

### CODE REVIEW

File: community/releases/index.md [hunk 1]
```diff
@@ -8,4 +8,6 @@ sidebar:
 * [Versions](/community/releases/versions/)
 * [Release Cadence](/community/releases/release-cadence/)
 * [Support Policy](/community/releases/support-policy/)
-
+* [Release Manager Runbook](/community/releases/manager-runbook/)
```
Comment (dorbin) [inline community/releases/index.md]:
```suggestion
* [Release Manager Runbook](/community/releases/release-manager-runbook/)
```

File: community/releases/patching-changelogs/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  nav: community
+---
+
+{% include toc %}
+
+It's not currently possible to submit a PR to adjust a changelog for a version
+of spinnaker that's already been released, as they're stored in gists.
+
+To change a [changelog](https://github.com/spinnaker/spinnaker.github.io/blob/master/_changelogs),
+please contact the Release Manager via [slack](http://join.spinnaker.io), on
```
Comment (dnilasor) [inline community/releases/patching-changelogs/index.md]:
```suggestion
please contact the Release Manager via [slack](http://join.spinnaker.io), in
```

File: community/releases/patching-changelogs/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+[#spinnaker-releases](https://spinnakerteam.slack.com/messages/spinnaker-releases/)
```
Comment (dnilasor) [inline community/releases/patching-changelogs/index.md]:
```suggestion
the [#spinnaker-releases](https://spinnakerteam.slack.com/messages/spinnaker-releases/) channel.
```

File: community/releases/patching-changelogs/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+It's not currently possible to submit a PR to adjust a changelog for a version
+of spinnaker that's already been released, as they're stored in gists.
+
+To change a [changelog](https://github.com/spinnaker/spinnaker.github.io/blob/master/_changelogs),
```
Comment (dnilasor) [inline community/releases/patching-changelogs/index.md]:
```suggestion
Release Managers are responsible for handling edits to existing changelogs. To request a change to a [changelog](https://github.com/spinnaker/spinnaker.github.io/blob/master/_changelogs),
```

File: community/releases/patching-changelogs/index.md (hunk 2 above)
Comment (dnilasor) [inline community/releases/patching-changelogs/index.md]:
```suggestion
please contact the Release Manager via [Spinnaker Slack](http://join.spinnaker.io), on
```

File: community/releases/patching-changelogs/index.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+please contact the Release Manager via [slack](http://join.spinnaker.io), on
+[#spinnaker-releases](https://spinnakerteam.slack.com/messages/spinnaker-releases/)
+to handle edits to existing changelogs.
```
Comment (dnilasor) [inline community/releases/patching-changelogs/index.md]:
```suggestion
```

**Review by dnilasor — CHANGES_REQUESTED** [review]
This looks fine to me. @brian-armory I'm not sure where exactly fits in the architecture of the new site (https://github.com/spinnaker/spinnaker.io - migration is in progress now and will be completed this year, for your reference @dbyron0), or if it makes more sense to add this as a section of a page rather than its own section of the nav. Regardless, seems like important information to share and I appreciate @dbryon0 for the PR! I made some minor suggestions for the language.

File: community/releases/patching-changelogs/index.md (hunk 2 above)
Comment (dbyron0) [reply community/releases/patching-changelogs/index.md]: https://github.com/spinnaker/spinnaker.github.io/pull/2031/commits/2796a64e735491e40469d993b26061b66fd0445b

**Comment by dogonthehorizon** [comment]
@mergifyio refresh

**Review by dogonthehorizon — APPROVED** [review]

**Comment by dogonthehorizon** [comment]
@mergifyio rebase

**Comment by dogonthehorizon** [comment]
@mergifyio update

**Comment by dogonthehorizon** [comment]
@dbyron0 would you mind rebasing off master? Should be good to merge when the build passes after that. Mergify doesn't seem to be playing nice this morning :(

**Comment by dbyron0** [comment]
@dogonthehorizon done

**Comment by dbyron0** [comment]
closes https://github.com/spinnaker/spinnaker.github.io/issues/2029

**Pull request merged**